dashboard/app/views/shared/_course_wide_block.haml

```diff
@@ -1,4 +1,6 @@
--if id == 'hoc'
+- showState ||= false
```

Comment:
this means showState is an optional parameter when including the partial, but doesn't hurt to specify it explicitly everywhere